Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/RayTracing/RayTracingSceneSrg.azsli

```diff
@@ -134,15 +134,13 @@ ShaderResourceGroup RayTracingSceneSrg : SRG_RayTracingScene
         uint     m_normalOffset;
         uint     m_tangentOffset;
         uint     m_bitangentOffset;
-        uint     m_uvOffset;        
-        float    m_padding0[2];
-
-        float4   m_irradianceColor;
-        float3x3 m_worldInvTranspose;
-        float    m_padding1;
-
+        uint     m_uvOffset;
+        
         uint     m_bufferFlags;
         uint     m_bufferStartIndex;
```

Comment:
not sure it is required since the following is a vector, but at least for good measures, a padding is required after 'm_bufferStartIndex'